src/angular/styles/_typography.scss

```diff
@@ -778,6 +790,10 @@
       --sbb-chips-background-color-disabled: var(--sbb-color-cloud);
       --sbb-chips-opacity: 0.9;
 
+      // Dialog
+      --sbb-dialog-overlay-backgroud-color: #{rgba(sbb.$sbbColorBlack, 0.7)};
```

Comment:
We already have a `// Dark Mode with Alpha` section. Please add `--sbb-color-black-alpha70` there and use it here (around line 122).